python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -115,87 +118,192 @@ def preprocess_ondisk_dataset(
     os.makedirs(os.path.join(dataset_dir, processed_dir_prefix), exist_ok=True)
     output_config = deepcopy(input_config)
 
-    # 2. Load the edge data and create a DGLGraph.
+    # 2. Load the data necessary to create a FusedCSCSamplingGraph.
     if "graph" not in input_config:
         raise RuntimeError("Invalid config: does not contain graph field.")
     is_homogeneous = "type" not in input_config["graph"]["nodes"][0]
     if is_homogeneous:
         # Homogeneous graph.
-        num_nodes = input_config["graph"]["nodes"][0]["num"]
         edge_fmt = input_config["graph"]["edges"][0]["format"]
         edge_path = input_config["graph"]["edges"][0]["path"]
         src, dst = read_edges(dataset_dir, edge_fmt, edge_path)
-        g = dgl.graph((src, dst), num_nodes=num_nodes)
+        num_nodes = input_config["graph"]["nodes"][0]["num"]
```

Comment:
this change makes the method body too large. pls wrap with function.

Reply:
any suggestions on https://github.com/dmlc/dgl/pull/6986/files#diff-d849f93b0af0c2a14419ea91a6c12bf81851c01425879f1cab7ae058c3fa2278R44 ?